modin/pandas/indexing.py

```diff
@@ -184,20 +168,27 @@ def _parse_tuple(tup):
     """
     Unpack the user input for getitem and setitem and compute ndim.
 
-    TODO: Add more details for this docstring template.
-
     loc[a] -> ([a], :), 1D
     loc[[a,b],] -> ([a,b], :),
     loc[a,b] -> ([a], [b]), 0D
 
     Parameters
     ----------
-    tup: tuple
-        [Descsription]
+    tup : tuple
+        User input to unpack.
 
     Returns
     -------
-    What this returns (if anything)
+    row_loc : list
+        List of row locators.
+    col_list : list
+        List of column locators.
+    ndim : {0, 1, 2}
+        Number of dimensions of located dataset.
+    row_scaler : bool
+        True of `row_loc` is a scalar, False otherwise.
+    col_scaler : bool
+        True of `col_loc` is a scalar, False otherwise.
```

Comment:
```suggestion
    row_scaler : bool
        True if `row_loc` is a scalar, False otherwise.
    col_scaler : bool
        True if `col_loc` is a scalar, False otherwise.
```